apps/private-law/prl-cron-hearing-actual-task/demo-image-policy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImagePolicy
+metadata:
+  name: demo-prl-cron-hearing-actual-task
```

Comment:
Hi, you would need to include this in https://github.com/hmcts/cnp-flux-config/blob/master/apps/private-law/automation/kustomization.yaml to use this new image policy